src/components/staking/LiValidator.vue

```diff
@@ -35,12 +35,12 @@
         :alt="`validator logo for ` + validator.name"
       />
       <div class="validator-info">
-        <h3 class="li-validator-name">
-          {{ validator.name }}
-        </h3>
-        <h3 class="li-validator-name li-validator-name-short">
+        <h3 v-if="validator.name.startsWith('0x')" class="li-validator-name">
           {{ validator.name | formatBech32 }}
```

Comment:
why not do this on the server so the FE stays agnostik?

Reply:
see @jbibla comment

Reply:
I can definitely move the `chooseFilter` thing to another PR. The `validator.website` improvement as well. Let's do it